client/config/crd/groupsnapshot.storage.k8s.io_volumegroupsnapshots.yaml

```diff
@@ -16,6 +16,15 @@ spec:
     - vgs
     singular: volumegroupsnapshot
   scope: Namespaced
+  conversion:
+    strategy: Webhook
+    webhook:
+      conversionReviewVersions: ["v1","v1beta1"]
+      clientConfig:
+        service:
+          namespace: default
+          name: snapshot-webhook-service
+          path: /convert
```

Comment:
If VGS v1beta1 is the same as v1beta2, I would not declare the conversion. The API server should do the right thing, without any code in the webhook

Reply:
Done.